monkey/monkey_island/cc/services/attack/technique_reports/T1021.py

```diff
@@ -0,0 +1,54 @@
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.services.attack.technique_reports.T1110 import T1110
+
+__author__ = "VakarisZ"
+
+
+class T1021(AttackTechnique):
+    tech_id = "T1021"
+    unscanned_msg = "Monkey didn't try to login to any remote services."
+    scanned_msg = "Monkey tried to login to remote services with valid credentials, but failed."
+    used_msg = "Monkey successfully logged into remote services on the network."
+
+    # Gets data about brute force attempts
+    query = [{'$match': {'telem_category': 'exploit',
+                         'data.attempts': {'$not': {'$size': 0}}}},
+             {'$project': {'_id': 0,
+                           'machine': '$data.machine',
+                           'info': '$data.info',
+                           'attempt_cnt': {'$size': '$data.attempts'},
+                           'attempts': {'$filter': {'input': '$data.attempts',
+                                                    'as': 'attempt',
+                                                    'cond': {'$and': [{'$eq': ['$$attempt.result', True]},
```

Comment:
why do you require SSH/password? Using hashes is also legitimate in this context.

Reply:
I followed mitre definition by the book. "An adversary may use Valid Accounts to log into a service specifically designed to accept remote connections, such as telnet, SSH, and VNC." Valid accounts means valid key/password, not PTH.

Reply:
Confirm that you want to change that and I'll change

Reply:
I don't see anything in the description which disqualifies hashes from this technique. Let's do it